.gitignore

```diff
@@ -0,0 +1,5 @@
+/grpc-service/target/
+/http-to-grpc/target/
+/project/project/
+/project/target/
+/target/
```

Comment:
Is there a reason to list all `*/target/` folders instead of _just_ [`target`](https://github.com/lagom/lagom-java.g8/blob/master/.gitignore#L1) ?

Reply:
I'm fine with a catch-all 'target' ignores, but I've been bitten by 'stray' 'target' directories causing trouble but not being obvious to find because they were caught by .gitignore in the past, so I usually create more targeted ones myself ;)